### PR TITLE
Milestone 1: Create classes

### PR DESCRIPTION
**Pull request by jasonfelice**
### School Library

- [x] Create `Person` class with `id`, `name`, and `age`.
- [x] Create `Student`  class that inherits from `Person`.
- [x] Create `Teacher` class that inherits from `Person`. 
- [x] Save each class in its separate file.

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown')
```
Comment (rustam-tolipov) [inline student.rb]: - [x] Kindly add `parent_permission` to your initializer and super.

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown')
```
Comment (rustam-tolipov) [inline teacher.rb]: - [x] The same as above you need to add `parent_permission` to your initializer and super.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @jasonfelice, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zhadier — APPROVED** [review]
## Hi @jasonfelice  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Optional
<hr/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**